lambdas/lambdas.mk

```
@@ -30,7 +30,7 @@ include $(abspath $(dir $(lastword $(MAKEFILE_LIST))))/../common.mk
 #
 .PHONY: compile
 compile: check_python
-	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor vendor/azul
+	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor vendor/azul vendor/humancellatlas
```

`vendor/azul` and `vendor/humancellatlas` are symlinks and as such are ignored by `compileall` when it encounters them during the traversal of `vendor`. Without the third positional argument, `compileall` never compiles the files in the directory that `vendor/humancellatlas` points to, and so `chalice package` compiles them on-the-fly later, without a pinned seed for `hash()`, producing non-deterministic byte code.

Instead of explicitly listing every symlink in `vendor` we should dynamically generate that list (untested):

```suggestion
	  PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor $(find vendor -maxdepth 1 -type l)
```

A short comment should explain why symbolic links need to be mentioned explicitly.